### ISSUE TITLE
NVDA doesn't notice all focus changes happening in Visual Studio 2013

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3818]
**Reported by MarcoZehe on 2014-01-27 05:21**
Steps to reproduce:
1. Start Visual Studio 2013.
2. Choose New Project, and in the templates, choose new XAML C# application.
3.Go to Project Explorer and find the Mainpage.xaml file.
4. Press Control+W, then x to go to the toolbox to add an UI element.

First result: Ctrl+W does not give any feedback.
1. Choose any element, for example a Textbox, and press Enter to add it to the UI.
2. Return to the UI designer with Shift+F7.

Second result: The fact that we're now in the designer does not get spoken.
1. Press Tab until you focus the textbox.
2. Press Alt+Enter for properties.

Expected: NVDA should announce that focus moved into the properties window. JAWS does this.
Actual: Only after you press Tab to force a focus change inside the newly focused Properties window will NVDA announce focus changes.

**Comment by leonardder** [comment]
@MarcoZehe, is this still of relevance to you?

**Comment by MarcoZehe** [comment]
No, I no longer use VS 2013.

**Comment by ehollig** [comment]
Closing, as this is no longer of relevance to the original author. If this is still an issue for someone in VS 2017, we can reopen this issue or a new issue can be created.
